sound/soc/sof/intel/hda-dsp.c

```diff
@@ -208,6 +208,21 @@ bool hda_dsp_core_is_enabled(struct snd_sof_dev *sdev,
 	return is_enable;
 }
 
+int hda_dsp_enable_core(struct snd_sof_dev *sdev, unsigned int core_mask)
```

Comment:
Generally f you have to number features in a patch commit message it means the patch needs split into more smaller patches